src/Expression/ForClasses/HaveNameMatching.php

```diff
@@ -15,10 +15,13 @@ class HaveNameMatching implements Expression
 {
     /** @var string */
     private $name;
+    /** @var array */
+    private $dependenciesToExclude;
 
-    public function __construct(string $name)
+    public function __construct(string $name, array $dependenciesToExclude = [])
```

Comment:
cosa centrano le dipendenze con HaveNameMatching? non basta l'except per questa regola?

Reply:
you are right @fain182  :)